cartoframes/data/dataset/registry/base_dataset.py

```diff
@@ -28,6 +30,7 @@ def __init__(self, credentials=None):
         self._table_name = None
         self._schema = None
         self._dataset_info = None
+        self._query = None
```

Comment:
Why is it needed here?

Reply:
It seems that this is not needed. Removing